src/libraries/src.proj

```diff
@@ -49,6 +49,9 @@
              Properties="$(TraversalGlobalProperties)" />
   </Target>
 
+  <Import Condition="'$(BuildTargetFramework)' == '$(NetCoreAppCurrent)'"
```

Comment:
nit: "`$(BuildNetCoreAppVertical)` == `true`"

Reply:
The `BuildNetCoreAppVertical` does not appear to be set at this point in the build.